tests/e2e/scale/noobaa/test_delete_objects.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+
+from datetime import datetime
+from ocs_ci.ocs import hsbench
+from ocs_ci.ocs.bucket_utils import (
+    s3_delete_objects,
+    list_objects_in_batches,
+    s3_delete_object,
+)
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id, scale
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="class")
+def s3bench(request):
+
+    s3bench = hsbench.HsBench()
+    s3bench.create_resource_hsbench()
+    s3bench.install_hsbench()
+
+    def teardown():
+        s3bench.cleanup()
+
+    request.addfinalizer(teardown)
+    return s3bench
+
+
+@scale
+class TestDeleteObjects:
+    @bugzilla("2181535")
+    @polarion_id("OCS-4916")
+    @pytest.mark.parametrize(
+        argnames=["delete_mode"],
+        argvalues=[
+            # uncomment the blow params when we have stable noobaa db
+            # or work around for memory issue caused during single deletion
+            # pytest.param("single"),
```

Comment:
It would be good to also add the BZ/issue tracking that issue here as a reference and to take an action to uncomment the code once the BZ is fixed

Reply:
@PrasadDesala I have added a comment mentioning what is the issue. There is no bz or issue pertaining to this.